spanner/spannertest/db_test.go

```diff
@@ -361,6 +362,83 @@ func TestConcurrentReadInsert(t *testing.T) {
 	}
 }
 
+func TestGeneratedColumn(t *testing.T) {
```

Comment:
Could we also add tests for:
1. Adding a generated column to an existing table.
2. Altering a generated column.
3. Dropping a column that is a dependant of a generated column.